packages/sdk-v2/src/sdk/buygd/utils/swapHelper.ts

```diff
@@ -0,0 +1,6 @@
+export const swapHelper = async (account: string) =>
+  fetch(`https://good-server.herokuapp.com/verify/swaphelper`, {
```

Comment:
dont hard code

Reply:
I don't see another way for now as it would break for a testing if I do take the URL based on env

Reply:
@L03TJ3 this is not a reason, always could add some flag to force using dev env on prod **in this specific feature**
so please add `useSwapHelper` hook which will accept `(account: string, env?: 'fuse' | 'development' | 'staging' | 'production')`. if env parameter is not set it gets value as it was shown in useFaucet example

Reply:
fixed now